Utils/GetFileExtension/README.md

```diff
@@ -0,0 +1,98 @@
+---
+topic: sample
+languages:
+- csharp
+products:
+- azure-cognitive-services
+name: Get File Extension sample skill for cognitive search
+description: This custom skill returns the document's extension and file name without extension.
+---
+
+# GetFileExtension
+
+This custom skill returns the document's file extension and the file name without extension to be indexed accordingly.
+
+## Requirements
+
+These skills have no additional requirements than the ones described in [the root `README.md` file](../../README.md).
+
```

Comment:
There should be a deployment section here (see other skills and use them as a template).

Reply:
Added :)